File: New/RootTools/src/com/rarnu/tools/root/RootApplication.java
package com.rarnu.tools.root;

import android.app.Application;

public class RootApplication extends Application {
	@Override
	public void onCreate() {
		super.onCreate();
		if (!GlobalInstance.DEBUG) {
			Thread.setDefaultUncaughtExceptionHandler(new RootUncaughtExceptionHandler(this));
		}
	}
}

File: RootTools2/src/main/kotlin/com/rarnu/tools/neo/RootApplication.kt
package com.rarnu.tools.neo

import android.app.Application
import java.util.*

/**
 * Created by rarnu on 12/5/16.
 */
class RootApplication : Application() {

    companion object {
        var isZh = false
    }

    override fun onCreate() {
        super.onCreate()
        isZh = Locale.getDefault().language == "zh"
    }
}